### PR TITLE
[9.0][MIG][survey_percent_question]

### PR DESCRIPTION
**Pull request by tarteo**

### CODE REVIEW

**Comment by tarteo** [comment]
Oh, let me check the unit tests.

**Comment by tarteo** [comment]
Unit tests are done!

File: survey_percent_question/i18n/survey_percent_question.pot [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+#: view:survey.question:survey_percent_question.survey_question_form
+msgid "{'invisible': [('type', 'not in', ['simple_choice', 'multiple_choice', 'matrix', 'percent_split'])]}"
+msgstr ""
+
+#. module: survey_percent_question
+#: code:addons/survey_percent_question/models/survey_question.py:64
+#, python-format
+msgid "{0} is not a number"
+msgstr ""
+
```
Comment (astirpe) [inline survey_percent_question/i18n/survey_percent_question.pot]: .pot file can be removed, is not used anymore

File: survey_percent_question/README.rst [hunk 2]
```diff
@@ -0,0 +1,66 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
```
Comment (astirpe) [inline survey_percent_question/README.rst]: Please add the target attribute for the image, as per the [readme template](https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst)

File: survey_percent_question/README.rst [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+Known issues / Roadmap
+======================
+
+* Refilling form when navigated back to previous page doesn't work
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/200/issues>`_. In case of trouble, please
```
Comment (astirpe) [inline survey_percent_question/README.rst]: Link here must be:
`https://github.com/OCA/survey/issues`

**Review by astirpe — CHANGES_REQUESTED** [review]

**Comment by tarteo** [comment]
@astirpe I fixed the README

**Review by astirpe — APPROVED** [review]

**Review by pankk — APPROVED** [review]
Tested locally. I successfully created and viewed the results of a survey with percent questions.

**Pull request merged**